### ISSUE TITLE
Doesn't see all requests in focused tab when starting browser

### ISSUE THREAD

**Issue by theneo9** [gorhill/uMatrix#93]
Here are my rules for reddit.com http://i.imgur.com/ROLDcXJ.png
As you can see I'm blocking css from redditmedia.com.

I have chrome set to "continue where I left off" -If I restart my browser, and reddit is the focused tab, it looks like this: http://i.imgur.com/8Qa9gfi.png

And the css is loaded.

Since no css request is seen, it must have been loaded from cache or something right?

Here is the problem with that. I've tried clearing entire browser history + cache + etc several times. Somehow chrome gets the CSS even though uMatrix is supposed to block it. That is the issue i'm having. Please note, this may be the case for other requests as well, I mean why would it be for css only? 

To summarize: 
Open a fresh browser with no tabs open, browse to reddit.com -> no css loaded, request is noted as blocked in uMatrix, as per my settings.

Leave tabs open, restart browser, css is somehow loaded now, and no request seen in uMatrix.

**Comment by gorhill** [comment]
Duplicate of #56.
